compat/src/index.js

```diff
@@ -129,7 +129,15 @@ let Children = {
 function createElement(...args) {
 	let vnode = h(...args);
 
+	vnode.$$typeof = REACT_ELEMENT_TYPE;
```

Comment:
IIRC this means it's not possible to do this anymore:


```js
import { createElement } from 'preact';
import { isValidElement } from 'react-is';

isValidElement(<div />)
```

This seems important since CLI creates non-compat vnodes for all JSX.